cmd/kubeadm/app/preflight/checks.go

```diff
@@ -347,6 +348,7 @@ type InPathCheck struct {
 	mandatory  bool
 	exec       utilsexec.Interface
 	label      string
+	fix        string
```

Comment:
Might want to call this `suggestion|advice`, b/c your applying it in a generic way to all checks.